### DISCUSSION TITLE
pH7Builder v17.2 RC1 Released

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#1017] [Release Announcements]
Fantastic news! ☀️**pH7Builder RC-1** has now been released for testing purposes! 🧑‍💻

This version contains several bug fixes, giving better stability to v17.2.

You can now test this new RC version and [open an issue](https://github.com/pH7Software/pH7-Social-Dating-CMS/issues) if you find any bugs in order to provide quality assurance for the next stable version 💫

Enjoy! 🤩

<hr /><em>This discussion was created from the release <a href='https://github.com/pH7Software/pH7-Social-Dating-CMS/releases/tag/17.2.0-rc.1'>pH7Builder v17.2 RC1 Released</a>.</em>

**Comment by coolyou11** [comment]
In your next update is there any possible chance that this script could have a way to create extra new pages, so we can add more of our own content?
